dizoo/mujoco/config/ant_onppo_config.py

```diff
@@ -35,17 +32,12 @@
             clip_ratio=0.2,
             adv_norm=True,
             value_norm=True,
-            # for onppo, when we recompute adv, we need the key done in data to split traj, so we must
-            # use ignore_done=False here,
-            # but when we add key traj_flag in data as the backup for key done, we could choose to use ignore_done=True
-            # for halfcheetah, the length=1000
-            # ignore_done=True,
-            ignore_done=False,
+            ignore_done=False,  # when we recompute advantage, we need the key done in data to split trajectories
             grad_clip_type='clip_norm',
             grad_clip_value=0.5,
         ),
         collect=dict(
-            collector_env_num=collector_env_num,
+            collector_env_num=10,
```

Comment:
`collector_env_num` should be unused here

Reply:
solved